### DISCUSSION TITLE
Is the Skeleton component planned?

### DISCUSSION THREAD

**Discussion by stanlrt** [chakra-ui/ark#2822] [Ideas]
Hello,

I was wondering if it is planned to implement a Skeleton component? Seems like even Zag does not have one so far. Of course, in the meantime we can use other libraries, but first party support would be awesome

**Comment by segunadebayo** [comment]
Skeleton is more of a UI-only component. Doesn't involve any logic.

Check out a library like Park UI: https://park-ui.com/react/docs/components/skeleton
